src/js/components/Tag/Tag.js

```diff
@@ -47,7 +67,7 @@ const Tag = forwardRef(
       console.warn('Tag cannot combine "onClick" and "onRemove".');
     }
 
-    const removeProps = !theme.tag.remove.kind
+    const computedRemoveProps = !theme.tag.remove.kind
```

Comment:
seems like  better name for this would have "theme" in it.

Reply:
yeah I was wondering on the naming thank you!

Reply:
I changed to `themeTagRemoveProps`